tests/01_03_52-disable_lock_whitelist.js

```diff
@@ -1,121 +1,78 @@
-    await wait(1000);
+    const federationAddressBalanceInitial = Number(await btcTxHelper.getAddressBalance(federationAddress));
 
     // address is not whitelisted
-    await test.assertLock(addresses, [{ address: federationAddress, amount: INITIAL_RSK_BALANCE }], { fails: true });
-    // wait for the btc to come back so we can use the assertLock method again
-    await utils.waitForBtcToReturn(addresses.btc);
-
-    const addr = await rskClient.eth.personal.importRawKey(WHITELIST_CHANGE_PK, '');
-    expect(addr.slice(2)).to.equal(WHITELIST_CHANGE_ADDR);
-    await rskClient.eth.personal.unlockAccount(addr, '');
-    // can disable the whitelist
-    await utils.sendTxWithCheck(
-      rskClient.rsk.bridge.methods.setLockWhitelistDisableBlockDelay(200),
-      (disableResult) => expect(Number(disableResult)).to.equal(1),
-      WHITELIST_CHANGE_ADDR)();
-
-    // disable whitelist doesn't work the second time
-    await utils.sendTxWithCheck(
-      rskClient.rsk.bridge.methods.setLockWhitelistDisableBlockDelay(10),
-      (disableResult) => expect(Number(disableResult)).to.equal(-1),
-      WHITELIST_CHANGE_ADDR)();
-
-    await btcClient.generate(100);
-    await wait(500);
-    await rskClient.fed.updateBridge();
-    await rskUtils.mineAndSync(rskTxHelpers);
-    await wait(500);
-
-    // address is still not able to send btc to bridge after 100 blocks
-    await test.assertLock(addresses, [{ address: federationAddress, amount: INITIAL_RSK_BALANCE }], { fails: true });
-    await utils.waitForBtcToReturn(addresses.btc);
-
-    await btcClient.generate(100);
-    await wait(500);
-    await rskClient.fed.updateBridge();
-    await rskUtils.mineAndSync(rskTxHelpers);
-    await wait(500);
-
-    // after 200 blocks the whitelist period has ended and we can send money to the bridge
-    await test.assertLock(addresses, [{ address: federationAddress, amount: INITIAL_RSK_BALANCE }]);
+    await sendPegin(rskTxHelper, btcTxHelper, btcAddressInfo, MINIMUM_PEGIN_VALUE_IN_BTC);
+    // wait for the btc to come back so we can use the sendPegin method again
+    await rskUtils.triggerRelease(rskTxHelpers, btcTxHelper);
+
+    // disable whitelisting after 20 blocks
+    const unlocked = await rskUtils.getUnlockedAddress(rskTxHelper, WHITELIST_CHANGE_PK, WHITELIST_CHANGE_ADDR);
+    expect(unlocked).to.be.true;
+    const disableLockWhitelistMethod = bridge.methods.setLockWhitelistDisableBlockDelay(20);
+    const disableResultCallback = (disableResult) => expect(Number(disableResult)).to.equal(1);
+    await rskUtils.sendTxWithCheck(rskTxHelper, disableLockWhitelistMethod, WHITELIST_CHANGE_ADDR, disableResultCallback);
+
+    await btcTxHelper.mine(10);
+    await rskUtils.waitAndUpdateBridge(rskTxHelper);
+
+    // address is still not able to send btc to bridge after 10 blocks
+    await sendPegin(rskTxHelper, btcTxHelper, btcAddressInfo, MINIMUM_PEGIN_VALUE_IN_BTC);
+    await rskUtils.triggerRelease(rskTxHelpers, btcTxHelper);
```

Comment:
Same assertions as before here

Reply:
hmm , to me it seems like too many assertions which are not directly related to this TC.  but ok... :)